sites/docs/docs/guides/usql-migration-guide.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Universal SQL Migration Guide
+sidebar_position: 3
+---
+
+Universal SQL is Evidence's latest release, and fundamentally changes how Evidence queries data in order to bring support for three big new features:
+1. **Multiple Data Sources:** Combine data from multiple databases, flat files like CSVs, and even non-SQL data sources like Google Sheets - and query them all with SQL
+2. **Inputs and Filters:** Use input components to dynamically update your queries and charts based on user input. Filter data using parameterized queries rather than Javascript filter syntax
+3. **Adapter Plugins:** Create your own adapter to enable connecting to any data source
+
+This is a big release and contains several breaking changes. This guide will help you migrate your existing projects to USQL.
+
+
+## Breaking Changes
+- Existing data source connections will need to be reconfigured
+- Queries to your data source will need to be moved to the new `sources` directory (details below)
+- Queries in markdown files and in the `queries` directory are now written in DuckDB SQL
+- Deprecated syntax:
+    - `{data.my_query}`. Use `{my_query}` instead
+    - Declaring variables from queries in `<script>` tags using `let my_var = ` or `var my_var = `. Use `$: my_var` instead
+
+## How Universal SQL Works
+- Universal SQL introduces a new data loading step (called `run sources`) - you write queries against multiple data sources, and Evidence will combine them all into Parquet files within a DuckDB WebAssembly instance - so you can query across them using SQL
+- Once your sources are loaded, they can be queried using DuckDB SQL syntax, either directly on your markdown page, or by writing a `.sql` file in your `queries` directory and referencing it in the frontmatter of your markdown file
+- These DuckDB SQL queries run in your browser and can be changed on the fly by interactive user input (e.g., filters, dropdowns) using parameterized queries (e.g., `where customer_id = '${inputs.customer}'`)
+
+## Change to Project Structure
+To support multiple data sources, USQL introduces a new `sources` directory, which contains one folder per data source. Each data source folder inclues connection configuration files (defined in YAML, but configurable via the settings UI of your project in your browser).
+
+If you already have a `sources` directory in your project (e.g., for `.sql` files or `.csv` files), it will need to be renamed to `queries`. This functions in the same way as before, but the `.sql` files use DuckDB SQL syntax and the queries run in the browser vs. during the build step.
```

Comment:
```suggestion
If you already have a `sources` directory in your project (e.g., for `.sql` files or `.csv` files), it will need to be renamed to `queries`. This functions in the same way as before, but the `.sql` files use DuckDB SQL syntax and the queries run in the browser rather than during the build step.
```

`rather than` might read a bit easier than vs